src/index.js

```diff
@@ -7,7 +7,7 @@ const FixedSizeChunker = require('./chunker-fixed-size')
 const through2 = require('through2')
 const UnixFS = require('ipfs-unixfs')
 const async = require('async')
-
+var isNode = !global.window
```

Comment:
actually we won't need to test the environment if we aren't going to use the file or dir importer in the browser. Will remove